instructions/bundling-scripts/zimbra-store.sh

```diff
@@ -599,6 +600,15 @@ CreateRhelPackage()
     	${repoDir}/zm-build/${currentScript}.spec
     echo "" >> ${repoDir}/zm-build/${currentScript}.spec
     echo "%clean" >> ${repoDir}/zm-build/${currentScript}.spec
+
```

Comment:
I would recommend not to modify zimbra-store package. 

Ideally this schema version should be part of package that container jars from zm-mailbox repositories. We have another task to create separate package from zimbrastore.jar zimbracommon.jar  and ...  We can take care as part of that task or even in future patches. 


Right now as part of this task create a file with ldap version and update attribute on ldap install/upgrade will be enough. 

We may not require to ship zimbra-store package as part of patch.

Reply:
So you are saying this should be part of a dependency package instead of the actual package that needs the value?

Are we planning on reorganizing the packaging scheme?   I won't merge this until we resolve the final solution.